test/mocks/config.js

```diff
@@ -26,7 +26,6 @@ const ruralPaymentsAgency = {
 }
 
 export const setEndemicsAndOptionalPIHunt = ({ endemicsEnabled, optionalPIHuntEnabled }) => {
```

Comment:
probably worth renaming the function? and changing the arguements, as we arent enabling / disabling endemics anymore